<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Launching browser]]>
    </line>
    <line>
      <![CDATA[Actual and Expected are matched TC passed]]>
    </line>
    <line>
      <![CDATA[Failed TC Name is validateUserID]]>
    </line>
  </reporter-output>
  <suite started-at="2022-07-07T15:08:25 IST" name="Default suite" finished-at="2022-07-07T15:09:28 IST" duration-ms="63261">
    <groups>
    </groups>
    <test started-at="2022-07-07T15:08:25 IST" name="Default test" finished-at="2022-07-07T15:09:28 IST" duration-ms="63261">
      <class name="KiteTest.validatekiteUserID2ByPrListener">
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:KiteTest.validatekiteUserID2ByPrListener@52719fb6]" started-at="2022-07-07T15:08:25 IST" name="launchBrowser" finished-at="2022-07-07T15:09:17 IST" duration-ms="51941" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Launching browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="loginToKiteApp()[pri:0, instance:KiteTest.validatekiteUserID2ByPrListener@52719fb6]" started-at="2022-07-07T15:09:17 IST" name="loginToKiteApp" finished-at="2022-07-07T15:09:18 IST" duration-ms="1308" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToKiteApp -->
        <test-method signature="validateUserID()[pri:0, instance:KiteTest.validatekiteUserID2ByPrListener@52719fb6]" started-at="2022-07-07T15:09:18 IST" name="validateUserID" finished-at="2022-07-07T15:09:20 IST" duration-ms="2517" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at KiteTest.validatekiteUserID2ByPrListener.validateUserID(validatekiteUserID2ByPrListener.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Actual and Expected are matched TC passed]]>
            </line>
            <line>
              <![CDATA[Failed TC Name is validateUserID]]>
            </line>
          </reporter-output>
        </test-method> <!-- validateUserID -->
        <test-method is-config="true" signature="logoutFromeKite()[pri:0, instance:KiteTest.validatekiteUserID2ByPrListener@52719fb6]" started-at="2022-07-07T15:09:21 IST" name="logoutFromeKite" finished-at="2022-07-07T15:09:25 IST" duration-ms="3869" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromeKite -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:KiteTest.validatekiteUserID2ByPrListener@52719fb6]" started-at="2022-07-07T15:09:25 IST" name="closeBrowser" finished-at="2022-07-07T15:09:28 IST" duration-ms="2552" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- KiteTest.validatekiteUserID2ByPrListener -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
